apps/web/components/editor/editable-field.tsx

```diff
@@ -57,7 +58,7 @@ export default function EditableField(props: FieldPropsType) {
       <div
         hidden={props.hidden}
         ref={nodeRef}
-        className="absolute top-0 left-0 m-auto h-16 w-48 cursor-move select-none flex-row-reverse p-2 text-center text-lg font-bold opacity-80"
+        className={`absolute top-0 left-0 m-auto h-16 w-48 cursor-move select-none flex-row-reverse p-2 text-center text-lg font-bold opacity-80 ${props.className}`}
```

Comment:
Not sure if this was the right place to add custom className, please do lmk if I need to make changes to this :)